<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RRRBotApplicationClass</class>
 <widget class="QMainWindow" name="RRRBotApplicationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RRRBotApplication</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="GeneralTab">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>291</width>
          <height>141</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="ProcessInfoGroupBox">
           <property name="title">
            <string>Process Info</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="1" column="1">
             <widget class="QLabel" name="ProcessIdValue">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="ProcessIdLabel">
              <property name="text">
               <string>Process id: </string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="WindowHandleValue">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="WindowHandleLabel">
              <property name="text">
               <string>Window handle:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="ProcessHandleLabel">
              <property name="text">
               <string>Process handle:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="ProcessHandleValue">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="GetProcessInfoButton">
              <property name="text">
               <string>Get Process Info</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="ProcessNameEdit"/>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="ModulesComboBox"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="ModulesLabel">
              <property name="text">
               <string>Modules:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="InfoTab">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Info</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>291</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="PlayerInfoGroupBox">
           <property name="title">
            <string>Player info</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="0">
             <widget class="QLabel" name="PositionLabel">
              <property name="text">
               <string>Position:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="AngleLabel">
              <property name="text">
               <string>Angle:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="NameValue">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="AngleValue">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="PositionValue">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="NameLabel">
              <property name="text">
               <string>Name:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="GethererTab">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Gatherer</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3"/>
      </widget>
      <widget class="QWidget" name="OffsetsTab">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Offsets</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="LogTab">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Log</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="ConsoleTab">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Console</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" colspan="2">
         <widget class="QTextEdit" name="CommandOutputEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="ClearButton">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="LaunchCommandButton">
          <property name="text">
           <string>Launch</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="ConsoleInputEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>335</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionLoad_Config"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Config">
   <property name="text">
    <string>Load Config</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rrrbotapplication.qrc"/>
 </resources>
 <connections/>
</ui>
